code/modules/mob/living/silicon/ai/ai.dm

```diff
@@ -348,18 +348,16 @@
 			clear_fullscreen("remote_view", 0)
 
 /mob/living/silicon/ai/update_sight()
-	. = ..()
+	see_in_dark = initial(see_in_dark)
```

Comment:
Silly goober, you made your PR on the wrong branch

Reply:
i sure did; it's so over for us